lib/webvalve/manager.rb

```diff
@@ -49,6 +83,38 @@ def allowlisted_urls
 
     private
 
+    def explicitly_enabled?
+      ENABLED_VALUES.include?(ENV['WEBVALVE_ENABLED'])
+    end
+
+    def services_enabled_by_default?
```

Comment:
I wonder if in some earlier routine it would make sense to do something like this:

```ruby
def set_defaults!
  if Webvalve.env.in?(ALWAYS_ENABLED_ENVS)
    if ENV.key? 'WEBVALVE_SERVICE_ENABLED_DEFAULT'
      WebValve.logger.warn SERVICE_ENABLED_DEFAULT_WARNING
    end
    if ENV.key? 'WEBVALVE_ENABLED'
      WebValve.logger.warn WEBVALVE_ENABLED_WARNING
    end
    ENV['WEBVALVE_ENABLED'] = '1'
    ENV['WEBVALVE_SERVICE_ENABLED_DEFAULT'] = '0'
  else
    ENV['WEBVALVE_ENABLED'] ||= '0'
    ENV['WEBVALVE_SERVICE_ENABLED_DEFAULT'] ||= '1'
  end
end
```

And then inside of `Manager` there doesn't need to be a concept of "always" or "explicit" - it can more naively trust the two ENV vars to mean what they say they mean:

```ruby
def enabled?
  ENABLED_VALUES.include?(ENV['WEBVALVE_ENABLED'])
end

def allowing?
  enabled? && services_enabled_by_default?
end

def intercepting?
  enabled? && !services_enabled_by_default?
end
```